stl/inc/xmemory

```diff
@@ -804,6 +804,7 @@ public:
     }
 };
 
+#if _HAS_DEPRECATED_ALLOCATOR_VOID

```

Comment:
As I indicated before, I don't think `_HAS_DEPRECATED_ALLOCATOR_VOID` should exist since its presence is unobservable.

Reply:
Dropped that patch